Zero_engine.alp

```diff
@@ -20717,6 +20805,18 @@ import zeroPackage.ZeroAccumulator;]]></Import>
 						</InitialValue>
 					</Properties>
 				</Variable>
+				<Variable Class="PlainVariable">
+					<Id>1726238237441</Id>
+					<Name><![CDATA[v_districtHeatDelivery_kW]]></Name>
```

Comment:
Is dit echt nodig? Zelfde comment als over v_currentIndoorTemp_degC; dit soort duplicates die niet de 'ground truth' zijn zorgen vaak voor fouten. Liever geen state-variables van energyAssets dupliceren in de GridConnection. 

Waarvoor wil je deze variabele gebruiken? Waarom wil je 'm op de gridConnection zetten?

Reply:
Ja ik snap je punt. Ik zal nalopen of ze nergens gebruikt worden in berekeningen, dat zou zeker zo moeten zijn voor de nieuwe variabelen die ik heb teogevoegd. Voor sommige variabelen die al bestaan zoals indoorTemp is dat niet zo.  

Maar voor de variabelen die niet in de berekening gebruikt worden zoals districtHeatDelivery is het heel handig voor de modelleur om op de canvas te zien wat er aan gaande is. Anders wordt het bouwen en aanpassen van casussen en stuk irritanter. Maar voor de variabelen die niet gebruikt worden gaat jou argument niet op. 

Dus wat mij betreft is het een keuze: willen we het canvas heel funciotneel en schoon met het nadeel dat er feedback krijgen meer werk is, of richten we de canvas zo in zodat we ook een beetje zien wat er speelt met het nadeel dat de canvas wat rommeliger kan worden.